### DISCUSSION TITLE
Please update the "Installing converter for Video DownloadHelper" page

### DISCUSSION THREAD

**Discussion by storm49152** [aclap-dev/video-downloadhelper#211] [Help and Q&A]
### Your question

Hi. I'm unsure if this is the right place, but here goes...

Via a link in the conversion manual, I landed on this page: [https://www.downloadhelper.net/install-converter3.php](https://www.downloadhelper.net/install-converter3.php)

It references libav a number of times, with two links to their website: [https://libav.org/](https://libav.org/). However, "libav.org" does not resolve to an IP address anymore, and according to Wikipedia, libav is an abandoned project.

I'm on Linux, but am not an Ubuntu or similar user. I don't know what's in the Ubuntu packages, and I'm having trouble to verify if my installation is correct using this documentation. Since the referenced website is down: can you please update this page with relevant information for non-Ubuntu users?

### Addon & CoApp version and details

I currently do not have a question about the Addon or CoApp themselves; therfore I have nothing to share here...

**Comment by jcfrog** [comment]
Hi,
These are old issues, do you use the latest versions? https://github.com/aclap-dev/video-downloadhelper/wiki#1-the-extension
There are some more indications for Linux users: https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-not-recognized#ubuntu-and-flatpak--snap-based-systems

**Comment by storm49152** [comment]
FTR: I'm using the VDH 8.2.1.1 and CoApp 2.0.19, which yesterday was fully up-to-date. But I don't think that's relevant to my question. Maybe it's because I'm not native English, but I'm unsure about what you mean with "old issues". Do you mean the page I mentioned is old/deprecated? Or that it is an old issue that the page is outdated? Or ... ?

- I was searching why VDH downloaded a (DASH streaming) video entirely, but the video was garbled (mostly gray, with artifacts, every time I download it). So I wanted to figure out whether I'm missing some library or tool.
- I landed on the main page of the VDH website, and clicked "Converter manual" under "Quick Links" on the main page.
- On this page, under "How to install the converter ?", there is a "Follow this link" to https://www.downloadhelper.net/install-converter3.php.
- It is on that page where there are references to the libav.org website, which is non-functional.

So, at this time I'm not asking for help solving the garbled video issue; I only opened this ticket to hint that the page where I ended up, via I think a logical click trail, is pointing to a non-functioning website, so people may not find the information they need.

**Reply by GrampaWildWilly** [reply]
If you do open a separate problem to diagnose your problem, I believe we will discover that the content you are trying to download is protected by DRM.  To determine whether that's true, disable DRM support in your browser settings.  Then reload the web page where the video resides.  If you see a browser message at the top of the window about DRM content on the page, then that's the problem.  VDH has never, does not now, nor will it ever download content protected by DRM.

But that doesn't address the issue of references to libav on the VDH web site.  My observation is that the web site is woefully out of date & contains a great deal of obsolete content.  I don't know if this libav stuff is obsolete.  I do know that this forum (and previously, our old forum on Google Groups) is a better source of current information.

**Comment by storm49152** [comment]
I just tried and yes, it's DRM protected, so that problem is "solved". No need for a new ticket.

If the website is so much out of date, then it doesn't feel useful to keep this ticket open because it doesn't feel like it will help anything or anyone. Therefore, closing ticket.

Thanks!

**Reply by paulrouget** [reply]
I'll fix the website. That's indeed deprecated.
